docs/operate/control/api-keys.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "API keys"
+linkTitle: "API keys"
+weight: 50
+layout: "docs"
+type: "docs"
+description: "Run control logic on a machine."
+images: ["/general/code.png"]
+date: "2024-01-13"
+---
+
+API keys grant access to organizations, locations, and machines.
+
+To view all API keys in use for your organization and the locations and machines inside it, click on the organization dropdown in the top navigation bar and click on **Settings**.
+
+{{<imgproc src="/fleet/api-keys.png" resize="700x" declaredimensions=true alt="API Keys table">}}
+
+## Add an API key
+
+On your organization's page, click **Generate key** to generate a new key.
+Optionally, give the key a name.
+Click on the **Resource** menu and choose what organization, location, or machine you want the key to grant access to.
+If you select organization, your key grants access to all locations and machines contained within that organization.
+If you select location, your key grants access to all of the machines contained within that location.
+
+For **Role**, assign either an **Owner** or **Operator** role.
+See [Permissions](/manage/manage/rbac/) for information about the privilege each role entails at each resource level.
+
+## Change an API key's access
+
+To edit an API key, click on **Show details** in the key's row of the key table's **Resources** column.
+
+{{<imgproc src="/fleet/additional-details.png" resize="700x" declaredimensions=true alt="Additional details for a key">}}
```

Comment:
Do we care that this shows the ID without a box over it? Not sure why that would matter without the key but just wondering how much we care about setting precedents and whether readers will wonder whether we meant to include it.

Reply:
I meant a redaction box--a solid grey box over the ID. Fine to leave if you're okay with it; just wanted to point it out in case we don't want to do this.